Microsoft.Toolkit.Uwp.SampleApp/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
```

Comment:
Still investigating....
Furthermore there are issues with the Identity assembly (they will soon update the package)